docs/troubleshooting-guide/diagnostic-info.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Diagnostic information for troubleshooting
+---
+
+# Diagnostic information for troubleshooting
+
+To download diagnostic information, hit `Cmd/Ctrl + F1`.
+
+Select the info you want to include in the diagnostic JSON file. Options include:
+
+- Item definition (like information about a dashboard)
+- Browser error messages
+- All server error messages
+- All server logs
+- Server logs from the current user only
+- Metabase instance version information
+
+What data Metabase captures depends on the page you're on when you hit `Cmd/Ctrl + F1`.
+
+> Review the downloaded file before sharing it, as the diagnostic info may contain sensitive data.
+
+## Metabase server and console logs
+
+Metabase will log errors, both on the server and in the browser console, depending on where the error occurs, which can help you track down an issue. Administrators will have access to the server logs, and everyone with a browser can open the developer tools to see the console logs.
+
+**Accessing the Metabase server logs**: You can look for the logs that Metabase leaves on the server's file system (or wherever else you collect logs). If you're logged into Metabase with an Admin account, you can also access the logs by going to the top right of the screen and clicking on the **gear** icon > **Admin settings** > **Troubleshooting** > **Logs**. Check out [How to read the server logs][server-logs].
```

Comment:
Should the link definition for `server-logs` be moved from the other doc? I actually don't know how this works